introduction.md

```diff
@@ -0,0 +1 @@
+<h1> Hello I am a super human untent onm destroying the universe whahahahahahah</h1>
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.